### PR TITLE
Test strategies

### PR DESCRIPTION
**Pull request by nociale**
Added test to partially address #197.

### CODE REVIEW

File: R/strategies.R [hunk 1]
```diff
@@ -113,9 +113,9 @@ strategy_MAR <- function(pars_group, pars_ref, index_mar){
 #' @export
 strategy_JR <- function(pars_group, pars_ref, index_mar){
 
-    if(sum(index_mar) == length(pars_group$mu)) {
+    if(all(index_mar)) {
         return(pars_group)
-    } else if(sum(index_mar) == 0) {
+    } else if(all(!index_mar)) {
```
Comment (gowerc) [inline R/strategies.R:118]: We have no `else` block so is the `else if` needed ?

File: R/strategies.R [hunk 2]
```diff
@@ -149,9 +149,9 @@ strategy_CR <- function(pars_group, pars_ref, index_mar){
 #' @export
 strategy_CIR <- function(pars_group, pars_ref, index_mar){
 
-    if(sum(index_mar) == length(pars_group$mu)) {
+    if(all(index_mar)) {
         return(pars_group)
-    } else if(sum(index_mar) == 0) {
+    } else if(all(!index_mar)) {
```
Comment (gowerc) [inline R/strategies.R:154]: same comment about the `else if`

File: R/strategies.R [hunk 3]
```diff
@@ -179,9 +179,9 @@ strategy_CIR <- function(pars_group, pars_ref, index_mar){
 #' @export
 strategy_LMCF <- function(pars_group, pars_ref, index_mar){
 
-    if(sum(index_mar) == length(pars_group$mu)) {
+    if(all(index_mar)) {
         return(pars_group)
-    } else if(sum(index_mar) == 0) {
+    } else if(all(!index_mar)) {
```
Comment (gowerc) [inline R/strategies.R:184]: same comment about the `else if`

**Comment by gowerc** [comment]
lgtm

**Review by gowerc — APPROVED** [review]

File: R/strategies.R [hunk 4]
```diff
@@ -178,10 +178,10 @@ strategy_CIR <- function(pars_group, pars_ref, index_mar) {
 #' @rdname stratgies
 #' @export
 strategy_LMCF <- function(pars_group, pars_ref, index_mar) {
-
-    if (sum(index_mar) == length(pars_group$mu)) {
+
```
Comment (nociale) [inline R/strategies.R:181]:
```suggestion
```

File: R/strategies.R [hunk 5]
```diff
@@ -178,10 +178,9 @@ strategy_CIR <- function(pars_group, pars_ref, index_mar) {
 #' @rdname stratgies
 #' @export
 strategy_LMCF <- function(pars_group, pars_ref, index_mar) {
-
-    if (sum(index_mar) == length(pars_group$mu)) {
+    if(all(index_mar)) {
```
Comment (nociale) [inline R/strategies.R:181]:
```suggestion

    if(all(index_mar)) {
```

**Pull request merged**